src/api/admin.ts

```diff
@@ -312,7 +313,10 @@ export const createVenue_v2 = async (input: VenueInput_v2, user: UserInfo) => {
   );
 };
 
-export const updateVenue = async (input: VenueInput, user: UserInfo) => {
+export const updateVenue = async (
+  input: WithId<VenueInput>,
+  user: UserInfo
+) => {
```

Comment:
This fixes it for admin v1, but will the same issue not also exist in admin v2 with the `updateVenue_v2` function below? (and possibly any other functions that rely on this same 'derive id from name' behaviour)